### PR TITLE
Update zh-TW translations

### PR DESCRIPTION
**Pull request by auroursa**
This is a small translation PR, I made some adjustments to existing translations, and added new translation strings based on 1.79.

### CODE REVIEW

**Comment by auroursa** [comment]
Please help me check this out @KuwaLee @noeFly @snowleo208 @SciencesVirus @cirx1e

**Comment by SciencesVirus** [comment]
#: src/screens/Login/LoginForm.tsx:262
msgid "Check your email for a login code and enter it here."
msgstr ""
msgstr "查看寄送至你電子郵件地址的驗證碼並在此輸入。"—>"在此輸入寄送至你電子郵件地址的驗證碼。"

: src/view/screens/Settings/DisableEmail2FADialog.tsx:102
msgid "To disable the email 2FA method, please verify your access to the email address."
msgstr ""
msgstr "若要關閉電子郵件雙重驗證，請驗證您對電子郵件地址的存取權限。"—>"若要關閉電子郵件雙重驗證，請驗證你的電子郵件地址。"

#: src/view/screens/Settings/index.tsx:757
msgid "Opens modal for account deletion confirmation. Requires email code"
msgstr "開啟用於帳號刪除確認的彈窗。需要電子郵件驗證碼。"
msgstr "開啟用於帳號刪除確認的彈窗。需要電子郵件驗證碼"—>"開啟帳號刪除的確認彈窗。需要電子郵件驗證碼"

我稍微看了一下此次修改的內容，覺得應該要稍微修正用詞，如上~

**Comment by auroursa** [comment]
@SciencesVirus Thank you for your review. I've made some changes to the translation based on your suggestions. 
Also, I've unified the translation of "dialog" as "對話框".

**Comment by noeFly** [comment]
其他軟體的翻譯好像「user」大多是翻譯成「使用者」？各位覺得呢？

**Comment by auroursa** [comment]
It's a bit awkward because I just made changes here... If 「使用者」 is more commonly used, I'll roll back the changes.

**Comment by SciencesVirus** [comment]
我覺得使用者或是用戶皆可

**Comment by auroursa** [comment]
I referred to the Traditional Chinese translation of Mastodon, where both 「用戶」 and 「使用者」 are used, for example, 「使用者介面」 and 「用戶數」. However, 「用戶」 is used more frequently to translate "user". So, I'm inclined towards 「用戶」. (Sorry, I didn't find a clearer example from Twitter regarding the translation of "user").

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -5594,7 +5556,7 @@ msgstr "你的電子郵件地址尚未驗證。這是一個我們建議的重要
 
 #: src/view/com/posts/FollowingEmptyState.tsx:47
 msgid "Your following feed is empty! Follow more users to see what's happening."
-msgstr "你的跟隨訊息流是空的！跟隨更多使用者看看發生了什麼事情。"
+msgstr "你的跟隨訊息流是空的！跟隨更多用戶看看發生了什麼事情。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5559]:
```suggestion
msgstr "你的跟隨訊息流是空的！跟隨更多用戶來看看發生了什麼事情。"
```

**Review by cirx1e — COMMENTED** [review]
稍微修正了一些用詞

順帶一提，我注意到有些字詞仍使用簡體字元
建議可以使用繁簡轉換工具再做一次檢查

- L1238:「`内`」->「`內`」
- L1558:「帐户」->「帳戶」 or 「帳號」
- L1611, L3127, L5172, L5176, L5184:「信息」->「資訊」 
- L2020:「設置」->「設定」
- L2059, L2157, L4678:「托管」->「託管」
- L2603:「在帖子文本和话题标签中隐藏该词」->「在貼文內容和話題標籤中隱藏該字詞」
- L2607:「仅在话题标签中隐藏该词」->「僅在話題標籤中隱藏該字詞」
- L3127:「额」->「額」
- L3954:「默認」->「預設」
- L4357:「复」->「覆」

**Comment by auroursa** [comment]
@cirx1e Thanks for your help and thorough review, and I apologize for my mistake. And all suggestions have been fully adopted now.

File: src/locale/locales/zh-TW/messages.po [hunk 2]
```diff
@@ -4697,7 +4659,7 @@ msgstr "{screenDescription} 已被標記："
 
 #: src/components/moderation/ScreenHider.tsx:111
 msgid "This account has requested that users sign in to view their profile."
-msgstr "此帳號要求使用者登入後才能查看其個人資料。"
+msgstr "此帳號要求用戶登入後才能查看其個人資料。"
```
Comment (KuwaLee) [inline src/locale/locales/zh-TW/messages.po:4662]: 同上建議“使用者”

File: src/locale/locales/zh-TW/messages.po [hunk 3]
```diff
@@ -4801,36 +4763,36 @@ msgstr "這應該在以下位置創建一個域記錄："
...
+msgstr "此用戶未跟隨任何人。"
 
 #: src/view/com/modals/SelfLabel.tsx:137
 msgid "This warning is only available for posts with media attached."
 msgstr "此警告僅適用於附帶媒體的貼文。"
 
 #: src/components/dialogs/MutedWords.tsx:283
 msgid "This will delete {0} from your muted words. You can always add it back later."
-msgstr "這將從你的靜音詞中刪除 {0}，你隨時可以在稍後添加回來。"
+msgstr "這將從你的靜音字詞中刪除 {0}，你隨時可以在稍後添加回來。"
```
Comment (KuwaLee) [inline src/locale/locales/zh-TW/messages.po:4795]: 同上建議“文字”

File: src/locale/locales/zh-TW/messages.po [hunk 4]
```diff
@@ -4851,15 +4813,15 @@ msgstr "對話串偏好"
...
+msgstr "若要關閉電子郵件雙重驗證，請驗證你的電子郵件地址。"
 
 #: src/components/ReportDialog/SelectLabelerView.tsx:33
 msgid "To whom would you like to send this report?"
 msgstr "你希望向誰提交此檢舉？"
 
 #: src/components/dialogs/MutedWords.tsx:112
 msgid "Toggle between muted word options."
-msgstr "在靜音詞選項之間切換。"
+msgstr "在靜音字詞選項之間切換。"
```
Comment (KuwaLee) [inline src/locale/locales/zh-TW/messages.po:4824]: 同上建議“文字”

File: src/locale/locales/zh-TW/messages.po [hunk 5]
```diff
@@ -5277,7 +5239,7 @@ msgstr "你已看完了你跟隨的貼文。這是 <0/> 的最新貼文。"
 
 #: src/components/dialogs/MutedWords.tsx:203
 msgid "We recommend avoiding common words that appear in many posts, since it can result in no posts being shown."
-msgstr "我們建議避免新增在許多貼文中常用的詞彙，因為這可能導致沒有貼文可顯示。"
+msgstr "我們建議避免新增在許多貼文中常用的字詞，因為這可能導致沒有貼文可顯示。"
```
Comment (KuwaLee) [inline src/locale/locales/zh-TW/messages.po:5242]: 同上建議“文字”

File: src/locale/locales/zh-TW/messages.po [hunk 6]
```diff
@@ -5313,7 +5275,7 @@ msgstr "很抱歉，我們無法解析此列表。如果問題持續發生，請
 
 #: src/components/dialogs/MutedWords.tsx:229
 msgid "We're sorry, but we weren't able to load your muted words at this time. Please try again."
-msgstr "很抱歉，我們目前無法加載你的靜音詞。請稍後再試。"
+msgstr "很抱歉，我們目前無法加載你的靜音字詞。請稍後再試。"
```
Comment (KuwaLee) [inline src/locale/locales/zh-TW/messages.po:5278]: 同上建議“文字”

File: src/locale/locales/zh-TW/messages.po [hunk 7]
```diff
@@ -5606,7 +5568,7 @@ msgstr "你的完整帳號代碼將修改為 <0>@{0}</0>"
 
 #: src/components/dialogs/MutedWords.tsx:220
 msgid "Your muted words"
-msgstr "你的靜音詞"
+msgstr "你的靜音字詞"
```
Comment (KuwaLee) [inline src/locale/locales/zh-TW/messages.po:5571]: 同上建議“文字”

File: src/locale/locales/zh-TW/messages.po [hunk 8]
```diff
@@ -1234,7 +1210,7 @@ msgstr "捨棄草稿？"
 #: src/screens/Moderation/index.tsx:518
 #: src/screens/Moderation/index.tsx:522
 msgid "Discourage apps from showing my account to logged-out users"
-msgstr "鼓勵應用程式不要向未登入使用者顯示我的帳號"
+msgstr "鼓勵應用程式不要向未登入用戶顯示我的帳號"
```
[the remark this answers is not in the document]
Comment (cirx1e) [reply src/locale/locales/zh-TW/messages.po:1213]: 「防止」這個詞容易遭成誤導

這個選項類似於 robots.txt ([wiki](https://developer.mozilla.org/en-US/docs/Glossary/Robots.txt))
只能請求應用不要向未登入的使用者顯示帳號，而非禁止

[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:1213]: 此功能並不限於robots.txt，也可以避免在未登入的情況下通過官方客戶端搜尋用戶

另外“防止”，並不是“禁止”的意思，防止有避免…但是仍有可能會發生的語意

https://bsky.app/profile/bsky.app/post/3kftdjdm3hd23


Screenshot_20240428-161648
Screenshot_20240428-162255

File: src/locale/locales/zh-TW/messages.po [hunk 9]
```diff
@@ -257,7 +249,7 @@ msgstr "替代文字"
 
 #: src/view/com/composer/photos/Gallery.tsx:209
 msgid "Alt text describes images for blind and low-vision users, and helps give context to everyone."
-msgstr "替代文字為盲人和視覺受損的使用者描述圖片，並幫助所有人提供上下文。"
+msgstr "替代文字為盲人和視覺受損的用戶描述圖片，並幫助所有人提供上下文。"
```
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:252]: 或许"背景資訊"更好？

[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:252]: 我們好像沒這用法
上下文我不確定我們這邊有沒有
但語境是肯定有的

[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:252]: 阿 我發現用情境這個詞可能更好

**Comment by auroursa** [comment]
@KuwaLee Translations have been modified according to the suggestions (including some content outside of this PR to ensure consistency in translation).
By the way, "協助工具" is the translation adopted from the previous PR. Some mobile manufacturers and Microsoft use this translation, but Google tends to use "無障礙" more often. This seems to be a matter of preference.

**Comment by KuwaLee** [comment]
> @KuwaLee Translations have been modified according to the suggestions (including some content outside of this PR to ensure consistency in translation). By the way, "協助工具" is the translation adopted from the previous PR. Some mobile manufacturers and Microsoft use this translation, but Google tends to use "無障礙" more often. This seems to be a matter of preference.

TBH, our government use "無障礙" more often

File: src/locale/locales/zh-TW/messages.po [hunk 10]
```diff
@@ -3356,7 +3324,7 @@ msgstr "貼文已隱藏"
 #: src/components/moderation/ModerationDetailsDialog.tsx:97
 #: src/lib/moderation/useModerationCauseDescription.ts:99
 msgid "Post Hidden by Muted Word"
-msgstr "貼文因靜音詞彙設定而被靜音"
+msgstr "貼文因靜音文字設定而被靜音"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:3327]:
```suggestion
msgstr "貼文因靜音文字設定而被隱藏"
```

File: src/locale/locales/zh-TW/messages.po [hunk 11]
```diff
@@ -3508,7 +3476,7 @@ msgstr "推薦訊息流"
 
 #: src/view/com/auth/onboarding/RecommendedFollows.tsx:181
 msgid "Recommended Users"
-msgstr "推薦的使用者"
+msgstr "推薦的用戶"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:3479]:
```suggestion
msgstr "推薦用戶"
```

File: src/locale/locales/zh-TW/messages.po [hunk 12]
```diff
@@ -3623,7 +3585,7 @@ msgstr "檢舉帳號"
 
 #: src/components/ReportDialog/index.tsx:49
 msgid "Report dialog"
-msgstr "檢舉頁"
+msgstr "檢舉對話框"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:3588]:
```suggestion
msgstr "檢舉對話"
```
我不知這句會在哪裏見到，到底是在檢舉什麼？檢舉貼文還是其他東西？ :thinking: 
還是這是彈窗的標題？

File: src/locale/locales/zh-TW/messages.po [hunk 13]
```diff
@@ -3712,16 +3674,16 @@ msgstr "要求發佈前提供替代文字"
 
 #: src/view/screens/Settings/Email2FAToggle.tsx:53
 msgid "Require email code to log into your account"
-msgstr ""
+msgstr "需要電子郵件驗證碼才能登入你的帳戶"
 
 #: src/screens/Signup/StepInfo/index.tsx:69
 msgid "Required for this provider"
-msgstr "提供商要求必填"
+msgstr "託管服務供應商要求必填"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:3681]:
```suggestion
msgstr "必須填上託管服務供應商"
```

File: src/locale/locales/zh-TW/messages.po [hunk 14]
```diff
@@ -3712,16 +3674,16 @@ msgstr "要求發佈前提供替代文字"
...
 
 #: src/view/screens/Settings/DisableEmail2FADialog.tsx:168
 #: src/view/screens/Settings/DisableEmail2FADialog.tsx:171
 msgid "Resend email"
-msgstr ""
+msgstr "重新發送"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:3686]:
```suggestion
msgstr "重新發送電子郵件"
```
你覺得需不需要`電子郵件`這一詞語？我見在原本的翻譯有寫

File: src/locale/locales/zh-TW/messages.po [hunk 15]
```diff
@@ -3989,7 +3951,7 @@ msgstr "選擇你希望訂閱訊息流中所包含的語言。未選擇任何語
 
 #: src/view/screens/LanguageSettings.tsx:98
 msgid "Select your app language for the default text to display in the app."
-msgstr "選擇你應用程式中要顯示的默認文字的語言。"
+msgstr "選擇你應用程式中要顯示的預設文字的語言。"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:3954]:
```suggestion
msgstr "選擇你應用程式中的預設語言。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 16]
```diff
@@ -4781,15 +4743,15 @@ msgstr "這則貼文已被刪除。"
 #: src/view/com/util/forms/PostDropdownBtn.tsx:370
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:250
 msgid "This post is only visible to logged-in users. It won't be visible to people who aren't logged in."
-msgstr "這則貼文僅對登入使用者可見。 未登入的人將看不到它。"
+msgstr "這則貼文僅對登入用戶可見。 未登入的人將看不到它。"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:4746]:
```suggestion
msgstr "只有登入用戶能見到這則貼文，未登入的人將看不到它。"
```
改得自然少許

File: src/locale/locales/zh-TW/messages.po [hunk 17]
```diff
@@ -4781,15 +4743,15 @@ msgstr "這則貼文已被刪除。"
...
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:352
 msgid "This post will be hidden from feeds."
 msgstr "這則貼文將從訊息流中被隱藏。"
 
 #: src/view/com/profile/ProfileMenu.tsx:370
 msgid "This profile is only visible to logged-in users. It won't be visible to people who aren't logged in."
-msgstr "此個人資料僅對登入使用者可見。 未登入的人將看不到它。"
+msgstr "此個人資料僅對已登入用戶可見。 未登入的人將看不到它。"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:4754]:
```suggestion
msgstr "只有登入用戶能見到此個人資料。 未登入的人將看不到它。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 18]
```diff
@@ -4851,15 +4813,15 @@ msgstr "對話串偏好"
 
 #: src/view/screens/Settings/DisableEmail2FADialog.tsx:102
 msgid "To disable the email 2FA method, please verify your access to the email address."
-msgstr ""
+msgstr "若要關閉電子郵件雙重驗證，請驗證你的電子郵件地址。"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:4816]: 這裏是不是叫用戶檢查電郵以關閉雙重驗證？
`verify your access`又未必指叫用戶再驗證電郵(用戶不是早已驗證了嗎？)，這裏似乎令人混淆

File: src/locale/locales/zh-TW/messages.po [hunk 19]
```diff
@@ -5095,70 +5057,70 @@ msgstr "使用者："
...
 #: src/view/com/modals/UserAddRemoveLists.tsx:196
 #: src/view/screens/ProfileList.tsx:777
 msgid "User list by you"
-msgstr "你的使用者列表"
+msgstr "你的用戶列表"
 
 #: src/view/com/modals/CreateOrEditList.tsx:197
 msgid "User list created"
-msgstr "使用者列表已建立"
+msgstr "用戶列表已建立"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:5095]:
```suggestion
msgstr "已建立用戶列表"
```

File: src/locale/locales/zh-TW/messages.po [hunk 20]
```diff
@@ -5095,70 +5057,70 @@ msgstr "使用者："
...
 
 #: src/view/com/modals/CreateOrEditList.tsx:183
 msgid "User list updated"
-msgstr "使用者列表已更新"
+msgstr "用戶列表已更新"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:5099]:
```suggestion
msgstr "已更新用戶列表"
```

File: src/locale/locales/zh-TW/messages.po [hunk 21]
```diff
@@ -5277,7 +5239,7 @@ msgstr "你已看完了你跟隨的貼文。這是 <0/> 的最新貼文。"
 
 #: src/components/dialogs/MutedWords.tsx:203
 msgid "We recommend avoiding common words that appear in many posts, since it can result in no posts being shown."
-msgstr "我們建議避免新增在許多貼文中常用的詞彙，因為這可能導致沒有貼文可顯示。"
+msgstr "我們建議避免新增在許多貼文中常用的文字，因為這可能導致沒有貼文可顯示。"
```
Comment (snowleo208) [inline src/locale/locales/zh-TW/messages.po:5242]:
```suggestion
msgstr "我們建議避免新增在許多貼文中常用的文字，因為這可能令你看不到任何貼文。"
```
這樣會自然少許，原文雖然沒有「你」，但我覺得這是官方的親切提示(註︰「我們」)，我覺得這樣寫也沒問題，你覺得如何？

**Review by snowleo208 — COMMENTED** [review]
Thanks for your translations! I added some minor comments to the PR, feel free to have a look or ignore the one you think it's not translated correctly.

謝謝你再次提供翻譯，我改了少許字眼。如你覺得翻得不太對的話可以不管 :smile:

File: src/locale/locales/zh-TW/messages.po [hunk 22]
```diff
@@ -485,7 +477,7 @@ msgstr "Bluesky"
 
 #: src/view/com/auth/server-input/index.tsx:154
 msgid "Bluesky is an open network where you can choose your hosting provider. Custom hosting is now available in beta for developers."
-msgstr "Bluesky 是一個開放的網路，你可以自行挑選託管服務提供商。現在，開發者也可以參與自訂託管服務的測試版本。"
+msgstr "Bluesky 是一個開放的網路，你可以自行挑選託管服務供應商。現在，開發者也可以參與自訂託管服務的測試版本。"
```
[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:480]: 我比較傾向保留“的”

File: src/locale/locales/zh-TW/messages.po (hunk 8 above)
[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:1213]: https://github.com/bluesky-social/social-app/pull/3678#discussion_r1582037688

可以看下這個

File: src/locale/locales/zh-TW/messages.po (hunk 11 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:3479]: 我傾向保留“的“

File: src/locale/locales/zh-TW/messages.po (hunk 18 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:4816]: 應該是在要關閉2AF之前說的
你要先驗證email才能繼續
？？？？

File: src/locale/locales/zh-TW/messages.po [hunk 23]
```diff
@@ -257,7 +249,7 @@ msgstr "替代文字"
 
 #: src/view/com/composer/photos/Gallery.tsx:209
 msgid "Alt text describes images for blind and low-vision users, and helps give context to everyone."
-msgstr "替代文字為盲人和視覺受損的使用者描述圖片，並幫助所有人提供上下文。"
+msgstr "替代文字為盲人和視覺受損的用戶描述圖片，並有助於為每個人提供背景資訊。"
```
[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:252]: 我覺得可以用“視障”

File: src/locale/locales/zh-TW/messages.po (hunk 9 above)
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po]: 情境确实感觉更好

File: src/locale/locales/zh-TW/messages.po (hunk 23 above)
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:252]: 替代文字為盲人和視障人士描述圖片及提供情境。

File: src/locale/locales/zh-TW/messages.po (hunk 22 above)
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:480]: 如果把 “open network”看成是一个名词，例如 Open Networking Foundation 開放網路基金會，确实可以删除「的」，但这里语境似乎不是这样，open 用于形容后面的 network，所以我也觉得保留「的」比较好

File: src/locale/locales/zh-TW/messages.po [hunk 24]
```diff
@@ -668,15 +660,15 @@ msgstr "來看看一些推薦的訊息流吧。點擊 + 將它們新增到你的
 
 #: src/view/com/auth/onboarding/RecommendedFollows.tsx:186
 msgid "Check out some recommended users. Follow them to see similar users."
-msgstr "來看看一些推薦的使用者吧。跟隨人來查看類似的使用者。"
+msgstr "來看看一些推薦的用戶吧。跟隨他們來查看類似的用戶。"
```
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:663]: 这里「的」删去确实能简洁一些

File: src/locale/locales/zh-TW/messages.po (hunk 8 above)
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:1213]: 我倾向先保留，因为目前其他地方也在还用“不要”，单独改可能不太合适
或许需要单独开个讨论出来，然后把其他地方也一起改掉，不然我感觉一部分用“不要”，一部分用“别”会比较割裂

File: src/locale/locales/zh-TW/messages.po (hunk 12 above)
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:3588]: 可能翻译成“檢舉彈窗”会更好
我在翻译这句话的时候往前查了下之前是怎么翻译“dialog”的，发现是“對話框”，就沿用了过来

File: src/locale/locales/zh-TW/messages.po (hunk 14 above)
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:3686]: 我把程式编译出来发现原本八个字可能太挤了，所以精简成四个字可能比较好看一点

File: src/locale/locales/zh-TW/messages.po (hunk 18 above)
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:4816]: 是的，你要先重新驗證email才能關閉 2FA 这个意思

File: src/locale/locales/zh-TW/messages.po (hunk 24 above)
[the remark this answers is not in the document]
Comment (auroursa) [reply src/locale/locales/zh-TW/messages.po:663]: 但我倾向保留后面一个“的”，不然我感觉语法会有点不正确

**Comment by auroursa** [comment]
@snowleo208 I have to say, many of your modifications look really good. There are a few that I'm keeping as they are for now. I've left comments below explaining why I didn't adopt some of the suggestions. If you feel my explanations aren't convincing, feel free to bring them up for discussion again.

**Comment by auroursa** [comment]
> > @KuwaLee Translations have been modified according to the suggestions (including some content outside of this PR to ensure consistency in translation). By the way, "協助工具" is the translation adopted from the previous PR. Some mobile manufacturers and Microsoft use this translation, but Google tends to use "無障礙" more often. This seems to be a matter of preference.
> 
> TBH, our government use "無障礙" more often for building or official documents 

OK we should follow the government's translation

File: src/locale/locales/zh-TW/messages.po (hunk 12 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:3588]: 那這樣應該不是“彈窗”而是“視窗”

File: src/locale/locales/zh-TW/messages.po [hunk 25]
```diff
@@ -1579,7 +1555,7 @@ msgstr "過多的提及或回覆"
 
 #: src/view/com/modals/DeleteAccount.tsx:230
 msgid "Exits account deletion process"
-msgstr "離開帐户删除流程"
+msgstr "離開删除帳號流程"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:1558]: 「删」-＞「刪」

```suggestion
msgstr "離開刪除帳號流程"
```

File: src/locale/locales/zh-TW/messages.po [hunk 26]
```diff
@@ -3712,16 +3674,16 @@ msgstr "要求發佈前提供替代文字"
 
 #: src/view/screens/Settings/Email2FAToggle.tsx:53
 msgid "Require email code to log into your account"
-msgstr ""
+msgstr "需要電子郵件驗證碼才能登入你的帳戶"
 
 #: src/screens/Signup/StepInfo/index.tsx:69
 msgid "Required for this provider"
-msgstr "提供商要求必填"
+msgstr "必須填上託管服務供應商"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:3681]:
```suggestion
msgstr "此供應商要求必填"
```

邀請碼欄位的備註
https://github.com/bluesky-social/social-app/blob/1dd3d6657c631905c27fa6af326d31a0f7c6039e/src/screens/Signup/StepInfo/index.tsx#L69

File: src/locale/locales/zh-TW/messages.po [hunk 27]
```diff
@@ -5313,7 +5275,7 @@ msgstr "很抱歉，我們無法解析此列表。如果問題持續發生，請
 
 #: src/components/dialogs/MutedWords.tsx:229
 msgid "We're sorry, but we weren't able to load your muted words at this time. Please try again."
-msgstr "很抱歉，我們目前無法加載你的靜音詞。請稍後再試。"
+msgstr "很抱歉，我們目前無法加載你的靜音文字。請稍後再試。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5278]: 「加載」-＞「載入」

```suggestion
msgstr "很抱歉，我們目前無法載入你的靜音文字。請稍後再試。"
```

**Comment by cirx1e** [comment]
抱歉，再補充幾個翻譯
L5250, L5254: 「加載」-＞「載入」
L5254: 「配置」-＞「設定」

謝謝！

**Comment by auroursa** [comment]
At the time, using ChatGPT for translation assistance was a wrong decision. It seemed unreliable and introduced many Simplified Chinese words...I'll stick to manual translation from now on.

**Comment by auroursa** [comment]
@cirx1e  By the way, I find the translation in L838 a bit strange. What do you think?

**Comment by cirx1e** [comment]
> @cirx1e By the way, I found the translation in L838 a bit strange. What do you think?

我認為可以翻譯成「調整 {name} (分類) 的內容過濾設定」
或「調整 {name} 分類的內容過濾設定」

**Comment by auroursa** [comment]
OK I think we're ready to merge.

**Review by pfrazee — APPROVED** [review]
Thanks so much everybody!

**Pull request merged**